tests/sampling/test_mcmc_external.py

```diff
@@ -17,16 +17,15 @@
 
 from pymc import Model, Normal, sample
 
-pytest.importorskip("nutpie")
-pytest.importorskip("blackjax")
-pytest.importorskip("numpyro")
-
 # turns all warnings into errors for this module
 pytestmark = pytest.mark.filterwarnings("error")
 
 
 @pytest.mark.parametrize("nuts_sampler", ["pymc", "nutpie", "blackjax", "numpyro"])
 def test_external_nuts_sampler(recwarn, nuts_sampler):
+    if nuts_sampler != "pymc":
+        pytest.importorskip(nuts_sampler)
```

Comment:
I checked this works as expected and skips only the parametrization that fails to import, not all of them afterwards. Can be seen in the CI as well: https://github.com/pymc-devs/pymc/actions/runs/4282342784/jobs/7456497043#step:8:79